### DISCUSSION TITLE
NET Maui DataSyncClientOptions HttpPipeline

### DISCUSSION THREAD

**Discussion by atexinspect** [Azure/azure-mobile-apps#824] [General]
Hi,
    I asked this question some time ago on Stackoverflow but did not get a working answer.

https://stackoverflow.com/questions/77284093/net-maui-datasyncclientoptions-httppipeline 

Any help appreciated.

Thanks
Paul.

**Comment by adrianhall** [comment]
The Stack Overflow answer is correct.

Your delegating handler added the XApiKey as a header - which is the proper thing to do as the TableController doesn't understand your XApiKey.  I'm not sure what else to say.  The TableController doesn't understand API keys.

**Reply by atexinspect** [reply]
Adrian,
           Thanks for the reply. The data is not sensitive so looks like I am stuck with unsecure API's. I will just remove the handler and
everything will then work again.

Regards
Paul.
